src/scss/component/_quickstart.scss

```diff
@@ -0,0 +1,46 @@
+.registry__container__sticky {
+	position: sticky;
+	top: 12px;
+
+	.o-labels {
+		@include oTypographyMargin($top: 2, $bottom: 3);
+	}
+}
+
+// Registry component version switcher.
+.registry-component-version-form {
+	margin-bottom: oTypographySpacingSize(2);
+	fieldset {
+		appearance: none;
+		border: 0;
+		padding: 0;
+		margin: 0;
+		min-width: 0;
+	}
+
+	legend {
+		@include oNormaliseVisuallyHidden();
+	}
+}
+
+
+.registry-link--github {
+	$color: str-slice(ie-hex-str(oColorsGetColorFor('link', 'text')), 4);
```

Comment:
Woah, I have no idea about what this is doing. I know this was moved from another file and not added in this PR but I would love to know what these functions are attempting to achieve

Reply:
It looks like this is ✨ magic ✨ for IE. 
[`ie-hex-str`](https://sass-lang.com/documentation/functions/color#ie-hex-str) returns a hex value for IE, but it includes the alpha channel, so 8 digits instead of 6.
[`str-slice`](https://sass-lang.com/documentation/functions/string#str-slice) keeps everything from the 4 index in the string onwards.

I think 🤔